test/DefaultCluster.Tests/GenericGrainTests.cs

```diff
@@ -245,7 +246,7 @@ public async Task Generic_SimpleGrain_GetGrain()
         [Fact, TestCategory("BVT"), TestCategory("Generics")]
         public async Task Generic_SimpleGrainControlFlow()
         {
-            var a = random.Next(100);
+            var a = ThreadSafeRandom.Next(100);
```

Comment:
Should just use `Random.Shared` (added in .NET 6) and eventually get rid of `ThreadSafeRandom` entirely.